### PR TITLE
Ubuntu 16.04 Compatibility updates

### PR DESCRIPTION
**Pull request by Datamancer1110**
Out of the box, Ubuntu 16.04 Xenial does not support installing PHP 5.5. I've updated the versions of the server packages in server_os_packages (specifically for Xenial) to reflect this, and I updated where on disk Icinga looks for the PHP.ini file which, in the case of Xenial using PHP5.6, is at /etc/php/5.6/apache2/php.ini

Because I need this to work in production, I've been applying my changes directly on top of 2.9.1, so I can't say for certain that this all works on the HEAD, but unless some very strange compatibility-breaking changes have been made since then, this code should work fine.

In short, without these changes, running the Icinga cookbook on Ubuntu 16.04 always fails. With these changes, it succeeds, at least for me.

### CODE REVIEW

**Comment by Datamancer1110** [comment]
By the way, this pull should fix https://github.com/Icinga/chef-icinga2/issues/240

**Comment by vkhatri** [comment]
LGTM, merging.

**Comment by vkhatri** [comment]
@Datamancer1110 👍
